path_2024.py

```diff
@@ -0,0 +1,78 @@
+"""
+Task 1 path.
+"""
+import pathlib
+import time
+
+import dronekit
+
+from modules import add_takeoff_and_landing_command
+from modules import upload_commands
+from modules import waypoints_to_commands
+from modules import waypoint_tracking
+
+
+WAYPOINT_FILE_PATH = pathlib.Path(".", "2023", "path_creation", "waypoints2024.csv")
+ALTITUDE = 40
+CONNECTION_ADDRESS = "tcp:localhost:14550"
+DELAY = 0.1  # seconds
+
+
+def run() -> int:
+    # Wait ready is false as the drone may be on the ground
+    drone = dronekit.connect(CONNECTION_ADDRESS, wait_ready = False)
+
+    # Read in hardcoded waypoints from csv files
+    waypoints = []
+    with open(WAYPOINT_FILE_PATH, encoding="utf-8") as file:
+        for line in file:
+            # Skip header and empty lines
+            parts = line.split(',')
+            if line in "name,latitude,longitude\n" or len(parts) < 3:
+                continue
+            name, latitude, longitude = parts
+            waypoint = [float(latitude), float(longitude)]
+            waypoints.append(waypoint)
```

Comment:
Call the CSV conversion function like in `path_2023.py` . Then, convert the dictionary into a list. Add a comment that the waypoints are expected to be in alphabetical order.